index.php

```diff
@@ -54,9 +92,11 @@
         снаряжение.</p>
       <ul class="promo__list">
         <!--заполните этот список из массива категорий-->
+        <?php foreach ($categories as $val):?>
```

Comment:
https://up.htmlacademy.ru/php/11/criteries#b3
```suggestion
        <?php foreach ($categories as $category):?>
```